### DISCUSSION TITLE
Is there a way to hide this message?

### DISCUSSION THREAD

**Discussion by niamleeson** [ibhagwan/fzf-lua#1133] [General]
<img width="307" alt="Screenshot 2024-04-13 at 5 34 31 PM" src="https://github.com/ibhagwan/fzf-lua/assets/5668478/12a89dc5-6b85-4b8d-979b-842f5aab74a9">

**Comment by ibhagwan** [comment]
`:FzfLua files no_header_i=true` or use `:FzfLua files header=false` the latter blanket hides all headers (even `cwd`, `search`, etc).

**Reply by cpcopy** [reply]
I tried such configuration with lsp_workspace_symbols. It looks like the `headers`, `no_header` and `no_header_i` only work when input as command parameters. It doesn't work when I put them into `lsp` global configuration. Is it expected?
```
        lsp = {
            no_header = true,
            headers = false,
            symbols = {
                symbol_style = 2,          -- style for document/workspace symbols, icon only
            }
        },      
```

**Reply by ibhagwan** [reply]
For symbols you’d need to put the setting under `lsp.symbols` (and `lsp.finder` for finder).
